include/libswiftnav/nav_msg.h

```diff
@@ -35,16 +35,26 @@ typedef struct {
    */
   s16 subframe_start_index;
 
-  u32 frame_words[3][8];
+  u32 frame_words[4][8];
   u8 next_subframe_id;
   s8 bit_polarity;
 
   u8 alert;
 } nav_msg_t;
 
+typedef struct {
+  ephemeris_t *ephemeris;
```

Comment:
Is there any reason ephemeris needs to be a pointer here now that we are passing around pointers to `gps_l1ca_decoded_data_t` ?